src/TripSegment/index.js

```diff
@@ -0,0 +1,329 @@
+  display: flex;
+  ${StyledText} {
+    line-height: 1.2;
+  }
+  &:hover {
+    ${StyledSegment} {
+      border-color: ${({ theme }) => theme.orbit.paletteCloudNormalHover};
+    }
+    ${Milestone}, ${Chevrons} {
+      svg {
+        color: ${({ theme }) => theme.orbit.colorIconPrimary};
+      }
+    }
+    ${ArrowDecor} {
+      &:before {
+        border-color: transparent ${({ theme }) => theme.orbit.paletteCloudNormalHover} transparent
+          transparent;
+      }
+    }
+  }
+`;
+
+StyledSegmentWrapper.defaultProps = {
+  theme: defaultTokens,
+};
+
+const WrapperCarrier = styled.div`
+  display: flex;
+  align-items: center;
+  position: relative;
+  left: ${({ theme }) => theme.orbit.spaceMedium};
+`;
+
+WrapperCarrier.defaultProps = {
+  theme: defaultTokens,
+};
+
+const WrapperInner = styled.div`
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  align-items: center;
+  padding: ${({ theme }) => theme.orbit.spaceXSmall};
+  cursor: pointer;
+
+  ${StyledCarrierLogo} {
+    padding: ${({ theme }) => `0 ${theme.orbit.spaceXXSmall} 0
+      ${theme.orbit.spaceXSmall}`};
+  }
+`;
+
+WrapperInner.defaultProps = {
+  theme: defaultTokens,
+};
+
+const WrapperColumn = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+const Time = styled.div`
+  padding: 0 ${({ theme }) => theme.orbit.spaceSmall} 0 0;
+`;
+
+Time.defaultProps = {
+  theme: defaultTokens,
+};
+
+export const ChildrenWrapper = styled.div`
+  width: 100%;
+  overflow: ${({ toggle }) => (toggle ? "visible" : "hidden")};
+  padding: 0 ${({ theme }) => theme.orbit.spaceXSmall};
+  max-height: ${({ visible, contentHeight, toggle }) =>
+    visible && (toggle ? `${contentHeight}px` : `0`)};
+  animation: ${({ toggle, theme, contentHeight, visible }) =>
+    visible &&
+    (toggle
+      ? `${theme.orbit.durationFast} ${ToggleDown(contentHeight)} linear`
+      : `${theme.orbit.durationFast} ${ToggleUp(contentHeight)} linear`)};
+
+  transition: max-height ${({ theme }) => theme.orbit.durationFast} linear,
+    padding ${({ theme }) => theme.orbit.durationFast} linear;
+  font-size: ${({ theme }) => theme.orbit.fontSizeTextSmall};
+  }
+`;
+
+ChildrenWrapper.defaultProps = {
+  theme: defaultTokens,
+};
+
+const ChildrenWrapperContent = styled.div`
+  border-top: 1px solid ${({ theme }) => theme.orbit.paletteCloudNormal};
+  padding: ${({ theme }) => theme.orbit.spaceXSmall} 0;
+  overflow: hidden;
+`;
+
+ChildrenWrapperContent.defaultProps = {
+  theme: defaultTokens,
+};
+
+const Icon = ({ type }) => {
+  switch (type) {
+    case "airline":
+      return <Airplane size="small" color="secondary" />;
+    case "train":
+      return <Train size="small" color="secondary" />;
+    case "bus":
+      return <Bus size="small" color="secondary" />;
+    default:
+      return <Airplane size="small" color="secondary" />;
+  }
+};
+
+class TripSegment extends React.PureComponent<Props, State> {
+  state = {
+    toggle: this.props.shown || false,
```

Comment:
Naming is hard i know, but this is little bit confusing.... 

please use `expanded` and `initialExpanded`